x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/fields_panel/fields_panel.tsx

```diff
@@ -78,51 +79,51 @@ export const FieldsPanel: FC<Props> = ({
   }
 
   return (
-    <EuiPanel data-test-subj={`mlDataVisualizerFieldsPanel ${fieldTypes}`}>
-      <EuiTitle>
-        <h2>{title}</h2>
-      </EuiTitle>
-      <EuiSpacer size="s" />
-      <EuiFlexGroup alignItems="center">
-        <EuiFlexItem grow={false}>
-          <span>
-            {showAllFields === true ? (
-              <FormattedMessage
-                id="xpack.ml.datavisualizer.fieldsPanel.showAllCountDescription"
-                defaultMessage="{wrappedCardsCount} {cardsCount, plural, one {field} other {fields}} ({wrappedPopulatedFieldCount} {populatedFieldCount, plural, one {exists} other {exist}} in documents)"
-                values={{
-                  cardsCount: fieldVisConfigs.length,
-                  wrappedCardsCount: <b>{fieldVisConfigs.length}</b>,
-                  populatedFieldCount,
-                  wrappedPopulatedFieldCount: <b>{populatedFieldCount}</b>,
-                }}
-              />
-            ) : (
-              <FormattedMessage
-                id="xpack.ml.datavisualizer.fieldsPanel.fieldsCountDescription"
-                defaultMessage="{wrappedCardsCount} {cardsCount, plural, one {field exists} other {fields exist}} in documents ({wrappedTotalFieldCount} in total)"
-                values={{
-                  cardsCount: fieldVisConfigs.length,
-                  wrappedCardsCount: <b>{fieldVisConfigs.length}</b>,
-                  wrappedTotalFieldCount: <b>{totalFieldCount}</b>,
-                }}
-              />
+    <div data-test-subj={`mlDataVisualizerFieldsPanel ${fieldTypes}`}>
+      <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+        <EuiFlexItem>
+          <EuiFlexGroup alignItems="center" gutterSize="m">
+            <EuiFlexItem grow={false}>
+              <EuiTitle size="m">
+                <h2>{title}</h2>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiBadge>
+                <b>{fieldVisConfigs.length}</b>
```

Comment:
I like the concise way that this number is now displayed, but it needs an accessible tooltip, displaying the contents that used to be shown in the label e.g. `7 fields exist in documents sampled`. Otherwise I don't think it is immediately obvious what it is showing.

Reply:
Fixed in https://github.com/elastic/kibana/pull/54358/commits/5a39b629f5283016edd7dbfeaadd57ba09dc54f8

Reply:
The `EuiTooltip` looks good. Is it possible to remove the `title` attribute from the badge, as you can end up with the default tooltip for '8' showing up as well as the `EuiTooltip`?
```
<span class="euiBadge euiBadge--iconLeft euiBadge--default" title="8"><span class="euiBadge__content"><span class="euiBadge__text"><b>8</b></span></span></span>
```

Reply:
https://github.com/elastic/kibana/pull/54358/commits/a153039edee19af13d2e089c4ec75d5d07827f82